test_files/test_config_all.yaml

```diff
@@ -8,14 +8,11 @@ metadata: all
 # The levels of taxa summary to include in the final summary of the analysis
 taxa_levels: all
 # The minimum abundance a taxon must have in a sample in order to be included in the legend
-abundance_threshold: 0.1
 # The size of the font in the plot legends
-font_size: 15
 # Columns to perform sub analysis on. 'all', 'none', or a comma seperated list
 # If an additional type of analysis should be run
-additional_analysis: sparcc
-sub_analysis: all
-# SparCC stuff
-iterations: 5
-permutations: 5
-type: spearman
+alpha_metrics: all
```

Comment:
where do the parameters for sparCC go?

Reply:
those SparCC parameters are example parameters from earlier MMEDS where you would specify additional analysis in this config file. Basically it was trying to say "after running standard analysis, also run SparCC". So that's no longer a valid example. But in terms of what about running SparCC otherwise, haven't gotten to that yet, will be part of a future PR